helm-charts/splunk-otel-collector/templates/operator/_instrumentation-spec.tpl

```diff
@@ -0,0 +1,38 @@
+{{- define "splunk-otel-collector.operator.instrumentation.spec" -}}
```

Comment:
I'd suggest we move this yaml to the `operator.instrumentation` section in values.yaml file. We already have pretty much the same snipped but commented out. If we move the actual default config there, it'll be easier for users to make change.

It won't work for the dynamic endpoints tho. We can keep them commented out in values.yaml and set them dynamically in the helper if users don't provide them explicitly.

Reply:
Done. I moved as much as possible to the value.yaml file within the spec section, this likely improves readability. Need to fix the github workflow for updating the java image.